ckanfunctionaltests/api/test_harvestobject.py

```diff
@@ -31,6 +31,6 @@ def test_harvestobject_html(inc_sync_sensitive, base_url, rsession, random_harve
         f"{base_url}/2/rest/harvestobject/{random_harvestobject_id}/html",
         allow_redirects=False,
     )
-    assert response.status_code == 301
+    assert response.status_code in [301, 302]
```

Comment:
Why has this status code changed? Could you elaborate in the commit message?

Reply:
I'll have to look up the code change in CKAN, probably mentioned in the changelog

Reply:
Change happened because of a switch to using flask which is using the CKAN helper, I think probably either response is ok. Have added the helper code which provides it.